jupyter-book/conditions/perturbation_modeling.ipynb

```diff
@@ -22,12 +22,12 @@
    "metadata": {},
```

Comment:
<p>I would structure this section (with the following headings) as:</p><ol><li>Motivation for identifying most responsive cell types</li><li>the Augur model</li><li class="ql-indent-1">Caveats for using Augur</li><li>Example: Predicting cell type prioritization for IFN-β stimulation</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>